configure/jaeger/README.md

```diff
@@ -2,13 +2,20 @@
 
 [Jaeger](https://github.com/jaegertracing/jaeger) is an open-source request tracing system that can run inside of your Kubernetes cluster.
 
-1.  Apply the Jaeger services to your cluster.
+1.  Append the `kubectl apply` command for the Jaeger resources to `configure/kubectl-apply-all.sh`.
 
     ```bash
+    # configure/kubectl-apply-all.sh
```

Comment:
same feedback for the rest and we talked in person, but rather make it a command to make this clearer

```
echo kubectl apply --prune -l deploy=jaeger -f configure/jaeger --recursive >> configure/kubectl-apply-all.sh
```